lib/opentracing/tracer.rb

```diff
@@ -24,12 +24,10 @@ def inject(span_context, format, carrier)
     end
 
     # Extract a span from a carrier
```

Comment:
Not a problem.  Not sure I understand what you're pointing out.  If you're referring to the method documentation, I believe I just copied the specification language.  If you want to fix in your own PR, by all means - no preference here.  Best, P